kujira/assetlist.json

```diff
@@ -2462,6 +2462,35 @@
           }
         }
       ]
+    },
+    {
+      "description": "The token of MCTest",
+      "denom_units": [
+        {
+          "denom": "factory/kujira1mn2pgkggquqm2m44zqlaq29jxwtad53qkv4xy4/MCTest",
+          "exponent": 0
+        },
+        {
+          "denom": "mc-test",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/kujira1mn2pgkggquqm2m44zqlaq29jxwtad53qkv4xy4/MCTest",
+      "name": "MCTEST",
+      "display": "mctest",
+      "symbol": "MC01",
+      "coingecko_id": "mctest-protocol",
```

Comment:
This coingecko-id does not exist yet in their API, please remove.
https://api.coingecko.com/api/v3/coins/list